src/app/components/feed/FeedPageContent.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { FormattedMessage, FormattedHTMLMessage } from 'react-intl';
+import { browserHistory } from 'react-router';
+import cx from 'classnames/bind';
+import AddIcon from '../../icons/add.svg';
+import FeedIcon from '../../icons/dynamic_feed.svg';
+import styles from './FeedPage.module.css';
+import Can from '../Can';
+import ButtonMain from '../cds/buttons-checkboxes-chips/ButtonMain';
+
+const FeedSection = ({
+  title, content,
+}) => (
+  <div className={cx(styles['feed-content-wrapper'], styles['feed-content'])}>
+    <h2 className="typography-subtitle2">
+      {title}
+    </h2>
+    <p className="typography-body1">
+      {content}
+    </p>
+  </div>
+);
+
+
+const FeedPageContent = ({ permissions, slug }) => {
+  const handleCreateFeed = () => {
+    browserHistory.push(`/${slug}/feed/create`);
+  };
+
+  return (
+    <>
+      <div className={styles['feed-page-header']}>
+        <div>
```

Comment:
@danielevalverde you don't really need this extra wrapping div, doesn't look like its doing anything really